### ISSUE TITLE
Updating plugin from jenkins

### ISSUE THREAD

**Issue by kacperleague9** [Osiris-Team/AutoPlug-Client#77]
**Problem:**
Recently you added server jar updater with jenkins support, but plugins can only be updated from bukkit and spigot.

**Solution:**
You could add very similar updater to server jenkins updater but for plugins by adding similar configuration to each plugin, useful for plugin beta build etc.

**Comment by Osiris-Team** [comment]
Oh if I would implement that a single plugins config in the plugins-config.yml would look like this:
```yaml
  WorldBorder: 
    exclude: false
    version: 1.9.10 (beta)
    latest-version: 
    author: Brettflan
    spigot-id: 60905
    bukkit-id: 0
    custom-author: 
    custom-check-url: 
    custom-download-url: 
    jenkins-projekt-url: https://some url
    jenkins-artifact-name: the artifact name
    jenkins-artifact-name-similarity: 70
    jenkins-build-id: 0
```
As you can see its getting pretty long and I don't think that It would affect and be useful for many plugins, because if they got a jenkins server they usually also update their plugins on spigot?
What do you say?

**Comment by kacperleague9** [comment]
not big deal, you can make it optional so by default not set at all (you can add a comment above the file that shows you example configuration with jenkins

**Comment by MaximMaximS** [comment]
true, I will really welcome this feature

**Comment by Osiris-Team** [comment]
Now available in 5.5
